io.github.cges30901.hmtimer.json

```diff
@@ -4,6 +4,9 @@
   "runtime-version": "5.9",
   "sdk": "org.kde.Sdk",
   "command": "hmtimer",
+  "rename-icon": "hmtimer",
+  "rename-desktop-file": "hmtimer.desktop",
+  "rename-appdata-file": "hmtimer.appdata.xml",
```

Comment:
So you moved these upstream but named them incorrectly?

Reply:
They should be named with the application’s ID, but I don't know how to rename it when executing `make install`(I tried something like install_icon64.extra but it doesn't work). Or maybe I can name those file directly in upstream and use names with the application’s ID even in non-flatpak build?

Reply:
Yes you should always be using the app-id for your files.